mlflow/genai/judges/make_judge.py

```diff
@@ -94,9 +94,9 @@ def _validate_feedback_value_type(feedback_value_type: Any) -> None:
 def make_judge(
     name: str,
     instructions: str,
-    feedback_value_type: Any,
```

Comment:
This argument has not been included in any release yet. Since this PR makes the arg optional prior to its initial release, we should place the arg add the end of the `make_judge` args list to avoid breaking changes.